docs/ruleset_functions/get_SWH_uses_associated_with_each_building_segment.md

```diff
@@ -18,7 +18,7 @@ Logic:
 - get the building segment: `building_segment = get_obj_by_id(RMD, building_segment_id)`
 - create a blank list: `swh_uses = []`
 - look at each swh use: `for swh_use in building_segment.service_water_heating_uses:`
-    - append the use to the list: `swh_uses.append(swh_use)`
+    - append the use to the list: `swh_uses.append(swh_use.served_by_distribution_system)`
```

Comment:
Why do we changed it to `served_by_distribution_system`? I thought the original implementation is pretty accurate?
If it is schema issue, then you will need to update the schema path to space in the for loop.

Reply:
Good point!

The reason was the [schema ](https://github.com/pnnl/ruleset-checking-tool/blob/RT/JG/ruletest_11_6/rct229/schema/ASHRAE229.schema.json#L4765) indicates that the ```served_by_distribution_system``` should match with the id of the ```ServiceWaterHeatingDistributionSystem``` not the id of the ```service_water_heating_uses```.

Could you check if this makes sense?

Reply:
Good point. Thank you for catching this up. I confused myself and followed the RDS.